cves/CVE-2015-5963.yml

```diff
@@ -127,10 +118,15 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: |
+    Unit tests did exist, but they did not test for the lack of a session upon logout,
```

Comment:
Make it a bit more clear who's doing what here. Is this malicious users? Development team? Bug testers?

Reply:
Doing what?